0-bootstrap/main.tf

```diff
@@ -103,6 +109,7 @@ module "cloudbuild_bootstrap" {
   version                     = "~> 2.1"
   org_id                      = var.org_id
   folder_id                   = google_folder.bootstrap.id
+  project_id                  = "prj-cicd"
```

Comment:
similar to above, this should be `prj-c-cicd`

Reply:
@rjerrems Considering the naming examples in the guide and the reason we added the `var.project_prefix` to prevent namespace collision, shouldn't the project be named as:

```terraform
"${var.project_prefix}-b-cicd-${random_id.suffix.hex}"

Reply:
@rjerrems  
 @daniel-cit raises a valid point here. I suspect not using `var.project_prefix` in the resource name can cause other conflicts.
 
Here's build logs from the previous commit that failed as well.
https://pantheon.corp.google.com/cloud-build/builds;region=global/78cd2aac-7927-4730-a9d0-5660ad4fd0b9;step=12?project=cloud-foundation-cicd

```
         91: resource "google_dns_policy" "default_policy" {
         on ../../../3-networks/envs/shared/dns-hub.tf line 91, in resource "google_dns_policy" "default_policy":
       
       Error: Error creating Policy: googleapi: Error 409: The resource 'entity.policy' named 'dp-dns-hub-default-policy' already exists, alreadyExists
```

Reply:
Yeah thanks for pickup @daniel-cit - I think we might need to add this back @z-shah . I discussed with @bharathkkb last week and we think that we should retain the project_prefix for now and we will try to make this more readable in a future release.